rasgotransforms/rasgotransforms/transforms/filter/filter.sql

```diff
@@ -10,7 +10,7 @@ SELECT *
 FROM {{ source_table }}
 {% for filter_block in items %}
 {%- set oloop = loop -%}
-{{ 'WHERE ' if oloop.first else ' AND ' }}
+{{ 'WHERE ' if oloop.first else ' ' ~ all_or_any ~ ' ' }}
```

Comment:
I was actually going to store this in the filter object itself so that way we can grab it from the filter object and we won't have to pass around a new variable